docker/keygen/files/signd-entrypoint

```diff
@@ -13,6 +13,7 @@ gpg: /usr/bin/gpg-copr
 allowuser: $(id -u)
 # we can't open a privileged (for the remote keygen server)
 allow-unprivileged-ports: true
+gnupghome: /var/lib/copr-keygen/gnupg/
```

Comment:
I have no objections against the change, but it would be nice to have a commit message explaining how this helps